src/games/yinsh.cc

```diff
@@ -199,7 +199,7 @@ tuple<int, int> StateForYinsh::find_first_invalid(int x,
   // printf("enter:    find_first_invalid\n");
 
   /// PENDING, shayad zaroorat nhi
-  int ex_x, ex_y;
+  int ex_x = -1, ex_y = -1;
```

Comment:
why?

Reply:
Avoid setting the nasal demons free. This was used without initializing before :(